langserver/references.go

```diff
@@ -54,6 +54,10 @@ func (h *LangHandler) handleTextDocumentReferences(ctx context.Context, conn JSO
 	}
 
 	if obj.Pkg() == nil {
+		if _, builtin := obj.(*types.Builtin); builtin {
+			// TODO: support builtin with textDocument/references
```

Comment:
agreed. Rather just say lets not support it in this comment due to number of references.